ui/apps/platform/src/css/trumps.css

```diff
@@ -5,7 +5,7 @@
  */
 
 .ReactModal__Content,
-#main-page-container > div > :not(.pf-c-page__main-section) {
+#main-page-container > div > :not(.pf-c-page__main-section):not(.pf-c-tab-content) {
```

Comment:
Fixes the text sizing when a PatternFly _Tab_ is the immediate first child of the main div, instead of a Patternfly PageSection.